client/scss/components/_page_description.scss

```diff
@@ -0,0 +1,66 @@
+//* @define page-description;
+
+.page-description {
+  &.page-description--wb {
+    @include respond-to('large') {
+      position: relative;
+      padding: 1.8rem 0 1.5rem span-columns('large', 1.5);
+
+      &:before {
+        content: '';
+        position: absolute;
+        top: 0;
+        bottom: 0;
+        left: 0;
+        width: span-columns('large', 0.5);
+        background: color('keyline-grey');
+      }
+    }
+  }
+
+  &.page-description--hn {
+    padding-bottom: 1.2rem;
+    border-bottom: 1px solid color('keyline-grey');
+  }
+}
+
+.page-description__title {
+  margin-top: 0;
+  margin-left: -0.06em;
+
+  &.page-description__title--wb {
+    @include font('WB3');
+    margin-bottom: 0.25em;
```

Comment:
This defines the text lock-up within the component – I don't think it can be abstracted away to universal font styling.